### ISSUE TITLE
Meteor 1.4.1_1 - No metadata files found for isopack

### ISSUE THREAD

**Issue by catin-black** [TAPevents/tap-i18n#183]
Hi I have a problem with update meteor to 1.4.1_1 because of tap-i18n

```
> meteor update
> Error: No metadata files found for isopack at: [...]/meteor/packages/tap_i18n/1.8.2
```

**Comment by catin-black** [comment]
I removed package from project update meteor and now on:

````
>meteor add tap:i18n
>[...].meteor\packages\meteor-tool\1.4.1_1\mt-os.windows.x86_32\isopackets\ddp\npm\node_modules\meteor\promise\node_modules\meteor-promise\promise_server.js:165
      throw error;
      ^
>Error: No metadata files found for isopack at: [...].meteor/packages/tap_i18n/1.8.2

**Comment by jkdowdle** [comment]
+1

**Comment by jkdowdle** [comment]
Ok I found a solution that worked for me, and I hope it will help you out. I will do my best to describe it.

I am on a Windows 10 machine. 

1 - Copy and paste all the packages names from <app>/.meteor/packages.
2 - Delete the apps .meteor folder. (<app>/.meteor)
3 - Uninstall Meteor completely
4- Go to C:/Users/(your user name)/AppData/Local/.meteor and again delete the .meteor folder 
5 - Reinstall Meteor 
6 - Make a new meteor application
7 - In this new meteor app, meteor add tap:i18n 
8 - copy and paste the .meteor folder from the new app into your original project. 
9 - at <app>/.meteor/packages copy and paste the list of all the packages from earlier. 
10 - Run the app and it will reinstall all those old packages again and it should work. 

I imagine that not all these steps are necessary (like uninstalling meteor) but this is just what I did. I also think it would work on a mac the same. Your meteor folders just may be located in an other place.
